lib/cmd/common/env.js

```diff
@@ -14,6 +14,13 @@ env.usage = "fhc env create <app-guid> <var_name> [<var_value>] --env=<environme
 var fhc = require("../../fhc");
 var common = require("../../common");
 var ini = require('../../utils/ini');
+var _ = require('underscore');
+
+var create = common.createAppEnv;
+var read = common.readAppEnv;
+var update = common.updateAppEnv;
+var remove = common.deleteAppEnv;
```

Comment:
Refactor redundant delegation function to alias. Should be equivalent, but reviewer attention needed.